test/elpa.el

```diff
@@ -0,0 +1,5 @@
+(setq package-user-dir
+      (expand-file-name (format ".cask/%s/elpa" emacs-version)))
```

Comment:
If the idea is to drop the cask dependency, why do we use a folder `.cask` something?